External/Plugins/FlashDebugger/Controls/DataTree/ValueNode.cs

```diff
@@ -34,20 +35,18 @@ public override string Value
                 string temp = null;
                 if (type == VariableType_.MOVIECLIP || type == VariableType_.OBJECT)
                 {
-                    string typeStr = "";
+                    string typeStr = m_Value.getTypeName().ToString();
+                    // rename vector
+                    typeStr = typeStr.Replace("__AS3__.vec.Vector.<", "Vector.<");
```

Comment:
You are completely right, copy & paste error. Well, if you decide everything else is right maybe you can merge it and then change the value to "__AS3__.vec::Vector.<". I cannot atm.